feedback/lib/src/better_feedback.dart

```diff
@@ -53,7 +95,7 @@ class BetterFeedback extends StatefulWidget {
   const BetterFeedback({
     Key? key,
     required this.child,
-    this.feedbackBuilder,
+    this.feedbackSheetBuilder,
```

Comment:
Can we try to not do a breaking change and instead just deprecate the old callback but keep it working?
Overall, this is just a visual / behavioral change, right?

Reply:
Yeah this was just visual, as I was working I kept getting confused as to what `feedbackBuilder` was. I'll snip this change from this PR-it belongs in it's own PR anyway (if we decide to do it).